assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -95,6 +95,12 @@ module.exports = elementorModules.ViewModule.extend( {
 	},
 
 	showModal: function( options ) {
+		const $closeButton = this.getModal().getElements( 'closeButton' );
+
+		this.buttons = [ $closeButton[ 0 ] ];
```

Comment:
Save the jQuery element (here and after). so it can be used on key down without re wrapping.
https://github.com/elementor/elementor/pull/11283/commits/609b023d4f7df93aba13252067ebdc58042acaf9#diff-9cdcc77155fc9dafa7f51b9526248e1fR619